pkg/fanal/artifact/remote/git_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux || bsd || darwin || freebsd
```

Comment:
We might be able to leverage `unix`.

```suggestion
//go:build unix
```
https://tip.golang.org/doc/go1.19#go-unix